frame/acronym.tex

```diff
@@ -1 +1,15 @@
-\newacronym{tk}{TK}{Techniker Krankenkasse}
\ No newline at end of file
+% Utility command to avoid duplicating the first value
+\newcommand{\newac}[2]{\newacronym{#1}{#1}{#2}}
```

Comment:
can we change this to 
\newcommand{\acr}[2]{
    \lowercase{\def\key{#1}}%
    \newacronym{\key}{#1}{#2}
}

that way the key will always be in lowercase? This may be useful in case there are acronyms with lowercase letters so for accessing those values you dont have to remember the exact casing

Reply:
That makes sense to me. I have added a comment that one can remove the lowercase bit (if one prefers that)